src/cozy/react/locales/en.json

```diff
@@ -58,13 +58,13 @@
   },
 
   "InstallationStep": {
-    "title": "Install Cozy extension",
+    "title": "Install Twake extension",
```

Comment:
Wondering how we will handle the transition as the extension has not been renamed yet, this may confuse the users.

Reply:
Good question, we only go in internal release for the moment, but we will need to synchronize for sure.